packages/viewer/src/types/loadableResource.ts

```diff
@@ -22,8 +27,31 @@ export function fromUrn(urn: string): LoadableResource {
 
   switch (resourceType) {
     case 'stream-key':
-      return { type: 'stream-key', id: resourceId };
+      return {
+        resource: { type: 'stream-key', id: resourceId },
+        query: fromQuery(uri.query),
+      };
     default:
       throw new Error('Invalid URN. Unknown resource type');
   }
 }
+
+export interface SceneViewStateResource {
+  type: 'scene-view-state';
+  id: string;
+}
+
+export type QueryResource = SceneViewStateResource;
+
+function fromQuery(query?: string): QueryResource | undefined {
+  if (query != null) {
+    const [resourceType, resourceId] = query.split('=');
+
+    switch (resourceType) {
+      case 'scene-view-state':
+        return { type: 'scene-view-state', id: resourceId };
+      default:
+        throw new Error('Invalid URN. Unknown query resource type');
+    }
+  }
+}
```

Comment:
Should we have something that's a bit more generic here? It's possible to have multiple query params.

Reply:
Yeah, I suppose we can add it here and just use the first query parameter during load (until we add support for starting from multiple scene-view-states if we want that in the future). I'll update this